source/win_evaluation.py

```diff
@@ -0,0 +1,48 @@
+from testing_base import *
```

Comment:
Maybe here instead of importing the testing_base, you could try directly importing the CardGames.py since it isn't a test. Other than that, the code looks awesome!